app/auth/routes.py

```diff
@@ -0,0 +1,95 @@
+"""
+Authentication settings for the BucketList API.
+
+Password verification and user registration takes place here.
+"""
+from flask import g, jsonify, request
+from flask_httpauth import HTTPBasicAuth
+
+from . import authe
+from app import errors
+from app.decorators import json
+from app.models import User
+
+auth = HTTPBasicAuth()
+
+
+@auth.verify_password
+def verify_password(token, password):
+    """
+    Verify token.
+
+    Verify token, password doesn't need to be present here. The token is
+    going to be in the request headers always.
+    """
+    user = User.verify_auth_token(token)
+
+    if not user:
+        return False
+
+    g.user = user
+    return True
+
+
+@authe.route('/login', methods=['POST'])
+def login():
+    """
+    Verify a user's identity.
+
+    Verify the user's identity and returns a token.
```

Comment:
Verify the user's identity is duplicated. The standard way to write a docstring can be found here:

1, https://www.python.org/dev/peps/pep-0008/#documentation-strings
2. https://www.python.org/dev/peps/pep-0257/